### PR TITLE
New threshold article

### PR DESCRIPTION
**Pull request by ahoneybun**
This adds a Threshold article to examine the new feature and it's current limitations for Open Firmware and Open EC systems.

### CODE REVIEW

File: _articles/battery.md [hunk 1]
```diff
@@ -57,19 +57,6 @@ The default values of 0 for the start threshold and 100 for the end threshold
 disable the feature.
 -->
```
Comment (crawfxrd) [inline _articles/battery.md:58]: This  commented out block isn't needed since the info is being moved.

Comment (ahoneybun) [reply _articles/battery.md:58]: So I should remove that sentence completely?

Comment (crawfxrd) [reply _articles/battery.md:58]: The whole thing. It's copied verbatim to the new article.

Comment (ahoneybun) [reply _articles/battery.md:58]: New article? This is the new article.

**Comment by jacobgkau** [comment]
@ahoneybun The current format of the article is:
```
## Some additional notes
### FlexiCharger (proprietary firmware)
### Charging Thresholds
#### Open Firmware
```
The additional note about the feature not being complete is only relevant to open firmware, and the explanation of charging thresholds is relevant to both open firmware and FlexiCharger. I'm going to try reorganizing the article like this instead:
```
Introduction
## Charging Thresholds (open firmware)
### Some additional notes
## FlexiCharger (proprietary firmware)
```

**Comment by jacobgkau** [comment]
> The larger the spread between the start and end charge percentages, the longer the physical battery will last. For optimal longevity, charge cycles of 40%~80% are a good idea.

Our "maximum lifespan" profile uses a range of 50%-60%. (The profiles were defined after research into thresholds and the current presets that other manufacturers use.) I'm going to remove/modify some of this info so as not to conflict with the presets and confuse people.

File: _articles/battery.md (hunk 1 above)
Comment (crawfxrd) [reply _articles/battery.md:58]: No, `battery.md` is the existing article. `laptop-battery-thresholds.md` is the new article.

Comment (jacobgkau) [reply _articles/battery.md]: I've taken care of this in https://github.com/system76/docs/pull/538/commits/7040adff30dd57c2b4c6fe1537a6671373689174.

**Review by jacobgkau — APPROVED** [review]
@ahoneybun Please make sure you're okay with how this reads now. If you are, then it should be good once https://github.com/system76/docs/pull/539 has been merged.

When the GUI is implemented, it should be easy to add an additional "Using the GUI:" section above the current two sections.

**Comment by ahoneybun** [comment]
Looks great @jacobgkau I have merged #539 so I will merge this one now.

**Pull request merged**